task-management/src/main/webapp/vue-app/tasks-management/components/tasks/TasksFilterDrawer.vue

```diff
@@ -405,6 +405,13 @@
           }
         });
       },
+      statusFilterLabel(item) {
+        if(item ==='All' || item ==='ToDo'|| item ==='InProgress'|| item ==='WaitingOn'|| item ==='Done') {
+          return this.$t(`label.status.${item.toLowerCase()}`);
```

Comment:
Could we simply remove the String **label.status.** if it is returned by the translation function this.$t ?

Reply:
if we remove it, the default status doesn't be translated